Taxes/Operation/01_ManageDecisionMatrix.md

```diff
@@ -0,0 +1,266 @@
+
+
+The correct order of the decision-making criteria is crucial, as the system goes through the criteria in the order they are listed, starting from the top, and stops checking them as soon as a matching criterion is found.
+This means, that the criteria must be organized from specific to general to cover all possible relevant cases.
+
+
+## Create an entry
+
+Create an entry to determine a new tax zone.
+
+#### Prerequisites  
+
+- An applicable tax class has been created, see [Create a tax class](../Integration/02_ManageTaxClasses.md#create-a-tax-class).
+- An applicable tax zone has been created, see [Create a tax zone](../Integration/03_ManageTaxZones.md#create-a-tax-zone).
+- An applicable tax rate has been created, see [Create a tax rate](../Integration/01_ManageTaxRates.md#create-a-tax-rate).
+
+#### Procedures
+
+*Taxes > Settings > Tab DECISION MATRIX*
+
+![Decision matrix](../../Assets/Screenshots/Taxes/Settings/DecisionMatrix/DecisionMatrix.png "[Decision matrix]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the upper left corner.   
+  A new data input line is displayed.
+
+2. Click the *VAT ID* drop-down list and select the appropriate option. In this case, the VAT ID refers to the buyer's VAT ID number. The following options are available:
+
+  - **Yes**  
+  Select this option if the customer is a company and, therefore, has a VAT ID number (B2B transaction).
+  - **No**  
+  Select this option if the customer is a private individual and, therefore, has no VAT ID number (B2C transaction).
+
+
+3. Click the *Country of origin* drop-down list and select the appropriate option. Use the drop-down list on the left to select a group of countries or the one on the right to select an individual country. As a group of countries, only the **EU - European Union** option is available. In the drop-down list of individual countries, all available countries in the system are displayed. When a group of countries is selected, the drop-down list on the right is locked.
+
+4. Click the *Ship-to country* drop-down list and select the appropriate option. Use the drop-down list on the left to select a group of countries or the one on the right to select an individual country. As a group of countries, only the **EU - European Union** option is available. In the drop-down list of individual countries, all available countries in the system are displayed. When a group of countries is selected, the drop-down list on the right is locked.
+
+5. If necessary, click the *Ship-to postcode* drop-down list and select the appropriate option. The following options are available:
+
+  - **Range**  
+  Select this option to specify a numeric postcode range, for example 27498 or 52001-52006.
+  - **Regex**  
+  Select this option to specify a postcode containing alphanumeric characters with regular expressions, for example /^BT.*/ in case of Northern Ireland. The site https://regex101.com/ can be useful to test regular expressions.
```

Comment:
```suggestion
  Select this option to specify a postcode containing alphanumeric characters with regular expressions, for example /^BT.*/ in case of Northern Ireland. The site https://regex101.com/ can be useful to test regular expressions.  
```